04 Event Emitters/assets/js/app/modules/Contacts.js

```diff
@@ -0,0 +1,78 @@
+(function ($, React, ReactDOM, App) {
+  'use strict';
+
+  var TableComponent = App.components.TableComponent;
+  var contactsService = App.contactsService;
+  var contacts;
+
+  var ContactsModule = (function () {
+
```

Comment:
I'm used to write module names with PascalCase, I will rename it. Should I only use that for constructor functions and namespaces? I'll take note of this!